### PR TITLE
change tender process to tender stage

### LINKED ISSUE #1514: Change "tender process" to "tender stage"
In a few places, such as `Tender.id`, `Tender.hasInquiries` and https://standard.open-contracting.org/staging/1.2-dev/en/schema/reference/#tender we speak about a "tender process". For consistency's and clarity's sake, I think we should only have two types of processes: planning and contracting. For activities linked to data published in `plannning`, `tender`, `award` and `contract`, I would stick to calling them "stages". Consequently, I would suggest replacing "tender process" with "during the tender stage". 

(We do not have "award processes" nor "contract processes", there we also only speak about stages.)

### PR DESCRIPTION
**Pull request by odscjen**
closes #1514 
~~Also added sentence about uniqueness of `tender.id` to it's description, the same sentence as appears in the descriptions of `contracts.id` and `awards.id`~~

### CODE REVIEW

File: docs/history/changelog.md [hunk 1]
```diff
@@ -201,6 +201,7 @@ Per the [normative and non-normative content and changes policy](https://docs.go
   * [#1189](https://github.com/open-contracting/standard/pull/1189) `Document.url`, to use precise wording and to describe how to indicate a part of a document.
   * [#1229](https://github.com/open-contracting/standard/pull/1229) Standardize the descriptions of `planning.documents`, `tender.documents`, `awards.documents`, `contracts.documents`, and `contracts.implementation.documents`.
   * [#1335](https://github.com/open-contracting/standard/pull/1335) Standardize the descriptions of `planning`, `planning.rationale`, `planning.budget`, `planning.documents`, and `planning.milestones`.
+  * [#1528](https://github.com/open-contracting/standard/pull/1528) `tender.id`, `tender.hasEnquires`, to reduce ambiguity and use consistent wording in description of procurement stages.
```
Comment (jpmckinney) [inline docs/history/changelog.md]:
```suggestion
  * [#1528](https://github.com/open-contracting/standard/pull/1528) `tender.id`, `tender.hasEnquiries`, to reduce ambiguity and use consistent wording in the description of procurement stages.
```

**Review by jpmckinney — COMMENTED** [review]
Why is that sentence added? Tender is singular. We have that sentence for awards and contracts because they are arrays (plural).

File: docs/history/changelog.md (hunk 1 above)
Comment (odscjen) [reply docs/history/changelog.md]: Ah, that explains why it wasn't there previously, I'll remove it.

**Comment by jpmckinney** [comment]
To close the issue, we also need to replace "tender process" in the second paragraph at https://standard.open-contracting.org/staging/1.2-dev/en/schema/reference/#tender

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**